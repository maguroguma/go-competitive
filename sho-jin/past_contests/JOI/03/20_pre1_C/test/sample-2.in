3 8
1 3 8
3 3 4 5 6 7 8 9
