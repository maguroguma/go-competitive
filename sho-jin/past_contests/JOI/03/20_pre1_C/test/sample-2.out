1
3
3
3
4
5
6
7
8
8
9
